apps/api/v2/src/modules/organizations/repositories/organizations-event-types.repository.ts

```diff
@@ -70,4 +70,26 @@ export class OrganizationsEventTypesRepository {
       include: { children: true },
     });
   }
+
+  async deleteUserManagedTeamEventTypes(userId: number, teamId: number) {
+    return this.dbWrite.prisma.eventType.deleteMany({
+      where: {
+        parent: {
+          teamId,
+        },
+        userId,
+      },
+    });
```

Comment:
managed event-types are the ones with a parentId that points to an event-type that belongs to the team